src/number_format.js

```diff
@@ -548,7 +579,13 @@ class NumberFormat extends React.Component {
     value = value.replace(/-/g, '');
 
     if (hasNegation && !removeNegation && allowNegative) {
-      value = '-' + value;
+      switch (negationFormat) {
```

Comment:
Instead of switch, this would be shorter. As there are only two value of negationFormat;
value = negationFormat === 'parentheses' ? `(${value})` : `-${value}`;
: